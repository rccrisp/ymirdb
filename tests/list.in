set a 1 2 3
snapshot
set b 4 5 6
set c a b
snapshot
list keys
list entries
list snapshots
checkout 1
list entries
checkout 2
list entries
forward c
rollback 1
list snapshots
bye
