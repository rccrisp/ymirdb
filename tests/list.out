> ok

> saved as snapshot 1

> ok

> ok

> saved as snapshot 2

> c
b
a

> c [a b]
b [4 5 6]
a [1 2 3]

> 2
1

> ok

> a [1 2 3]

> ok

> c [a b]
b [4 5 6]
a [1 2 3]

> a, b

> ok

> 1

> bye
